BookReader/BookReader.js

```diff
@@ -817,13 +827,13 @@ BookReader.prototype.drawLeafsOnePage = function() {
 
         if (BookReader.util.notInArray(indicesToDisplay[i], this.displayedIndices)) {
             var width = parseInt(this._getPageWidth(index)/this.reduce);
+            var leftMargin = parseInt((containerWidth - width) / 2);
```

Comment:
What are you using `parseInt` for here? Are you rounding / truncating?

Reply:
rounding to nearest digit, doesn't have to be exact, just enough.

Reply:
I might use `Math.round()` then to be more clear about the intention since `parseInt` is usually used for converting strings to ints.

Reply:
✔️

Reply:
@jason, this is following the same convention in the function.  I changed it in the plugin code to use `Math.round`.  With this one, do you want me to update all the instances in this function, too?

Reply:
confirmed with Jason that we can hold this change in core functionality for now.